packages/core/src/particle/ParticleGenerator.ts

```diff
@@ -253,6 +269,13 @@ export class ParticleGenerator {
     this._retireActiveParticles();
     this._freeRetiredParticles();
 
+    if (this.main.simulationSpace === ParticleSimulationSpace.World) {
+      this._retireActiveBounds();
+      this._updateBoundingBoxWorldSpace();
```

Comment:
You call  `this.isAlive` in function, this call timing is not the final result of this frame!